.github/workflows/main.yml

```diff
@@ -11,6 +11,22 @@ on:
   pull_request:
 
 jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          any-of-labels: "Status: Waiting for Customer"
```

Comment:
Maybe we could add the [`start-date`](https://github.com/actions/stale#start-date) parameter if we don't want current issues to be analyzed by this action.

Reply:
Added 'start-date' property. Not sure about the necessity of 'debug-only', since we anyway need to add a specific label for the flow to be triggered.

Reply:
Yes, the `debug-only` would be to just play around before actually changing issues status.
You could add the label and have this new workflow log what it would do so we could check that it would be what is expected.
Anyway, it was just an idea.

Reply:
added the property, let's see how it works